MTA-6811, 6812, 6813, 6814, 6815, 6816 CQA April Developer Lightspeed work (#357)
migtools/mta-documentation: assemblies/developer-lightspeed-guide/assembly_getting-started.adoc

:context: getting-started

[role="_abstract"]

> You can view prerequisites, persistent volume requirements, installation, and workflows that help you to decide how you want to use the {mta-dl-full}.

Reviewer: Short description should rather be structured as "Why, what". This is a user story format that we should use. Here's my suggestion:
```suggestion
Before installing and deploying the {mta-dl-full}, review the system requirements and deployment workflows.
```
